intTests/test_mir_setup_errors/test.rs

```diff
@@ -0,0 +1,9 @@
+// Non-void: used to test required mir_return, etc.
+pub fn id(x: u32) -> u32 {
+    x
+}
+
+// Void: used to test that mir_return is forbidden.
```

Comment:
I think this comment is out of date, as we do in fact have specs for `id_void` that use `mir_return`.